Makefile

```diff
@@ -45,7 +45,7 @@ workloads:
 
 .PHONY: integration-tests
 integration-tests: install-cross check-bins move-bins up pod-status-check workloads
-	cross test --release -- --nocapture
+	cargo test --release -- --nocapture
```

Comment:
What happens if we use cross now that we have the tools installed?

Reply:
I'll give it a try. Any reason we are testing with a release build?

Reply:
No particular reasons

Reply:
okay, I think now I understand why the integration tests failed in this repo, but not in the `containerd-wasm-shims` repo and why we were seeing the missing `protoc` even though we clearly installed it here: https://github.com/spinkube/containerd-shim-spin/blob/main/cross/Dockerfile#L20-L21. I should have noticed this earlier haha. 

When I extract the spin shim out from the containerd-wasm-shims repo, besides the obvious work to delete other shims, I've also decided to make the shim part of the root workspace, while the old spin shim is in it's own workspace (e.g. https://github.com/deislabs/containerd-wasm-shims/blob/main/containerd-shim-spin/Cargo.toml#L35). So the test library and shim in the deislabs repo are completely independent rust workspaces and have their own targets, this explains why `cross test` without `--target` works in the deislabs repo, but not here, and also explains why we were seeing the missing `protoc` because there is no `--target` defined in `cross test` command. 

I bet that if we make the spin shim as it's own workspace, the pipeline would succeed, but this probably isn't what we want. I think it's okay to just run `cargo test -p tests` instead of `cross test` for integration tests as we just want to run the test functions in [`tests` folder](https://github.com/spinkube/containerd-shim-spin/tree/main/tests).

Reply:
Sounds good.  As a note incase we run into it in the future, In Runwasi we needed to support cross for the e2e tests running in KIND because we needed a static binary since KIND containers didn't have the [glibc](https://github.com/containerd/runwasi/pull/356#issuecomment-1765382961) libraries for dynamic loading.  It looks like we use k3ds and don't have that issue here.  If we ever switch or want to use Kind for any reason we may need to reconsider.